core/src/test/java/org/apache/druid/java/util/emitter/core/HttpEmitterTest.java

```diff
@@ -70,6 +70,7 @@ public void timeoutEmptyQueue() throws IOException, InterruptedException
     final HttpEmitterConfig config = new HttpEmitterConfig.Builder("http://foo.bar")
         .setBatchingStrategy(BatchingStrategy.ONLY_EVENTS)
         .setHttpTimeoutAllowanceFactor(timeoutAllowanceFactor)
+        .setFlushTimeout(5000)
```

Comment:
To make it easier to adjust the timeout later if needed, perhaps have a common `HttpEmitterConfig.Builder` for the tests that sets the flush timeout to the appropriate value.

Or a named constant for the value, which can also be used for the new prop value added in `ParametrizedUriEmitterTest`.